server.py

```
@@ -0,0 +1,255 @@
+import asyncio
+import websockets
+import google.genai as genai
+from google.genai import types
+import json
+import os
+import base64
+import logging
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)